src/puptoo/process/profile.py

```diff
@@ -770,6 +772,18 @@ def system_profile(
                 rhel_ai_profile["nvidia_gpu_models"] = [gpu["model"] for gpu in nvidia_smi_l]
             profile["rhel_ai"] = _remove_empties(rhel_ai_profile)
 
+    if image_builder_facts:
+        ib_facts = {}
+        prof_id = image_builder_facts.get("image-builder.insights.compliance-profile-id")
+        if prof_id:
+            ib_facts["compliance_profile_id"] = prof_id
+        pol_id = image_builder_facts.get("image-builder.insights.compliance-policy-id")
+        if pol_id:
+            ib_facts["compliance_policy_id"] = pol_id
+
+        profile["image_builder"] = ib_facts
```

Comment:
@croissanne I'm wondering when this `ib_facts` got empty, do we need still populate this `image_builder` fact in system_profile? Shall we make a `if` check to avoid this case?

Reply:
sure